DashKitUI/PumpManager/DashHUDProvider.swift

```diff
@@ -55,8 +55,9 @@ internal class DashHUDProvider: NSObject, HUDProvider {
         return reservoirView
     }
 
-    public func didTapOnHUDView(_ view: BaseHUDView) -> HUDTapAction? {
-        return HUDTapAction.presentViewController(pumpManager.settingsViewController(bluetoothProvider: bluetoothProvider, colorPalette: colorPalette))
+    public func didTapOnHUDView(_ view: BaseHUDView, allowDebugFeatures: Bool) -> HUDTapAction? {
```

Comment:
[nit] should `allowDebugFeatures` default to `false`?

Reply:
`didTapOnHUDView` is a protocol method.